single_tech_samples/fabric/fabric_ci_cd/README.md

```diff
@@ -57,17 +57,20 @@ Refer to the [know issues, limitations, and workarounds](./issues-limitations-an
 
 Here are the steps to use the bootstrap script:
 
-1. Clone the repository:
+1. Clone the repository and get the subscription information for your environment variables:
 
     ```bash
     # Optional
     az login --tenant "<tenant_id>"
     az account set -s "<subscription_id>"
+    az account show
     cd "<installation_folder>"
     # Repo clone
     git clone https://github.com/Azure-Samples/modern-data-warehouse-dataops.git
     ```
 
+Note: While running the az login --tenant command you may be prompted for a number that refers to the Subscription ID. If you are prompted for this number then there is no need to run the az account set command again.
```

Comment:
this is not needed if you add `az config set core.login_experience_v2=off` before the az login command, az login will not try to log you onto all the subscriptions you have access to (thus making the login immediate, as it's not checking for subscriptions), then with the selection of the subscription (following command) you are selecting the subscription you need, so you are good.
So I suggest removing this comment and adding back the login experience flag